.github/workflows/markdownlint.yml

```diff
@@ -3,13 +3,13 @@ name: Markdownlint
 permissions:
   contents: read
 
-on:
-  pull_request:
-    paths:
-      - "**/*.md"
-      - ".markdownlint.json"
-      - ".github/workflows/markdownlint.yml"
-      - ".github/workflows/markdownlint-problem-matcher.json"
+# run even on changes without markdown changes, so that we can
+# make it in GitHub a required check for PR's
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - '*'
```

Comment:
I don't think this is valid GitHub Actions workflow syntax.

```suggestion
# run even on changes without markdown changes, so that we can
# make it in GitHub a required check for PR's
on:
  pull_request:
```

Reply:
thanks, I am blindly copying..